_posts/2017-08-13-gsoc-karl-1.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Contributing to Babel: Three GitHub Issues at a Time"
+author: "Karl Cheng"
+slug: contributing-to-babel-three-github-issues-at-a-time
+---
+
+Getting to work your way around a new code base always poses its challenges, and Babel was no exception.
+
+I’ve been working with Babel as part of the Google Summer of Code 2017 programme, working to update Babel transforms and the Babylon parser to accommodate changes to specifications and implementing new features.
```

Comment:
just Australian/british english 😛, depends on if we care about changing those, dono 😄